### PR TITLE
💁 Defaults are back

### PR DESCRIPTION
**Pull request by rowanseymour**
Re-adds default values for some things in the context. Things with no default still render like dicts.

| Object           | Default |
|--------------|---------|
|`contact` | name or urn or id |
|`results`| Name: value\n... same as `@flow` in legacy |
|`results.foo` | value |
|`input`| text\nattachments same as `@step.value` in legacy |
|`channel`| name |
|`flow`|  name |
|`test_result`|  true/false |
|`run`|  _none_ |
|`trigger`|  _none_ |
|`fields`| _none_ |
|`urns`|  _none_ |
|`legacy_extra`|  _none_ |

Not sure how I feel about the inconsistency but some things just don't have a natural default.

### CODE REVIEW

File: docs/functions.json [hunk 1]
```diff
@@ -511,21 +511,6 @@
             }
         ]
     },
-    {
-        "signature": "format_input(urn)",
```
Comment (rowanseymour) [inline docs/functions.json]: this function was definitely weird and annoying that it had to be the operand for all migrated rulesets on @step.value

**Comment by nicpottier** [comment]
Is address the right thing for channel? Perhaps name is more flexible? (think shortcodes where we have more than one channel with the same address)

File: docs/functions.json (hunk 1 above)
Comment (nicpottier) [reply docs/functions.json]: yes, win!

**Comment by rowanseymour** [comment]
@nicpottier `@channel` in legacy world is address... but I have no strong feelings here.. all feels a bit arbitrary... if it is name, maybe same is true for `@flow`...

**Comment by nicpottier** [comment]
Ya, name just feels most natural. Also maps to what we do for contact where name is the highest priority thing.

**Pull request merged**